### PR TITLE
Fixed: Every New Broadcast should be Broadcasted to Subscribers

### PR DESCRIPTION
**Pull request by BHS-Harish**
## Issue that this pull request solves

 Closes: #1125 

## Proposed changes

As soon as a broadcast gets approved by the admins, it should broadcast it to all the email subscribers. This will help students to be updated with any new opportunities posted on the site.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

image

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by BHS-Harish** [comment]
Hey @Kajol-Kumari 

Please review it and merge it.

The PR failed due to some failures but those are no affect our functionality

File: frontend/src/components/Footer/footer.module.scss [hunk 1]
```diff
@@ -703,7 +702,7 @@ a > span {
   }
 }
 
-@media screen and (max-width: 510px) {
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/footer.module.scss:706]: this change shouldn't have come here, make sure to pull the latest master, rest looks fine

**Review by Kajol-Kumari — APPROVED** [review]

File: frontend/src/components/Footer/footer.module.scss (hunk 1 above)
Comment (BHS-Harish) [reply frontend/src/components/Footer/footer.module.scss:706]: Sorry for that.

Comment (Kajol-Kumari) [reply frontend/src/components/Footer/footer.module.scss:706]: Its okay, we are all learning 😊

**Pull request merged**